src/index.html

```diff
@@ -11,7 +11,120 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <!-- <h1>HTML Form</h1>
+    <script type="text/javascript" src="./main.js"></script> -->
+
+    <form action="./main.js" method="post">
+      <fieldset class="box">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input type="text" name="surname" autocomplete="off">
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input type="text" name="name" autocomplete="off">
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input type="date" name="birthDay">
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement
+          <input type="checkbox" name="terms">
+        </label>
+      </fieldset>
+
+      <fieldset class="box">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input type="email" name="email" placeholder="email@example.com">
+        </label>
+
+        <label class="form-field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="6"
+            maxlength="24"
+            required>
+        </label>
+      </fieldset>
+
+      <fieldset class="box">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="form-field">
+          Do you love cats?
+          <input type="radio" name="loveCats" value="yes">Yes
+          <input type="radio" name="loveCats" value="no">No
```

Comment:
```suggestion
          <input type="radio" name="loveCats" value="yes">Yes
          <input type="radio" name="loveCats" value="no">No
```
Для цих двох інпутів мають бути також і два label, щоб збільшити зону клікабельності